integrationTests/testProcessorNode.go

```diff
@@ -533,7 +533,7 @@ func NewTestProcessorNodeWithFullGenesis(
 		BlockChainHook:    tpn.BlockchainHook,
 		BlockChain:        tpn.BlockChain,
 		ArwenChangeLocker: tpn.ArwenChangeLocker,
-		Bootstrapper:      tpn.Bootstrapper,
+		Bootstrapper:      syncDisabled.NewDisabledBootstrapper(),
```

Comment:
I would have set a disabled Bootstrapper as default on all test processor nodes in order to always have a non nil pointer for the tpn.Bootstrapper .
Also, I would have changed L2291 to this:
```GO
if fmt.Sprintf("%T", tpn.Bootstrapper) == "*disabledBootstrapper"{
    return errors.New("no bootstrapper available")
}
//.....
```

Reply:
done